cluster-operator/src/main/java/io/strimzi/operator/cluster/model/KafkaMirrorMakerCluster.java

```diff
@@ -454,14 +454,6 @@ public PodDisruptionBudget generatePodDisruptionBudget() {
         return PodDisruptionBudgetUtils.createPodDisruptionBudget(componentName, namespace, labels, ownerReference, templatePodDisruptionBudget);
     }
 
```

Comment:
No need for the double empty lines here which happen after you delete the method.